<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Qin maxima [L/hr]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="CaudalIN1"/>
            </item>
            <item>
             <widget class="QDial" name="Dial1"/>
            </item>
            <item>
             <widget class="QLCDNumber" name="Display1"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Capacidad [L]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Capacidad1"/>
            </item>
            <item>
             <widget class="QProgressBar" name="tanque1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="value">
               <number>24</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="enbprinc1">
              <property name="text">
               <string>Enable</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>33</width>
            <height>165</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Qout Maxima</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="caudalout1"/>
          </item>
          <item>
           <widget class="QDial" name="Dial2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="display2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Capacidad [L]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Capacidad0"/>
            </item>
            <item>
             <widget class="QProgressBar" name="Tanque0">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="value">
               <number>24</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="enbprinc0">
              <property name="text">
               <string>Enable</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Qin maxima [L/hr]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="CaudalIN0"/>
            </item>
            <item>
             <widget class="QDial" name="Dial0"/>
            </item>
            <item>
             <widget class="QLCDNumber" name="display0"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Qout maxima</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="caudalout2"/>
            </item>
            <item>
             <widget class="QDial" name="Dial3"/>
            </item>
            <item>
             <widget class="QLCDNumber" name="display3"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Capacidad [L]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="capacidad2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="Tanque2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="value">
               <number>24</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="enable1">
              <property name="text">
               <string>Enable</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Capacidad [L]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Capacidad3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="Tanque3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="value">
               <number>24</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="enable2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Enable</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Qout maxima [L/hr]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="caudalout3"/>
            </item>
            <item>
             <widget class="QDial" name="dial4"/>
            </item>
            <item>
             <widget class="QLCDNumber" name="display4"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
